src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leader Board</title>
+</head>
+<body>
+    <header>
+        <h1>Leader Board</h1>
+    </header>
+    <main>
+        <section class="table">
+            <div class="recent-scores">
+                <h3 >Recent scores</h3>
+                <button type="button" id="refresh">Refresh</button>
+            </div>
+            <div class="scores">
+                <p>Name: 100</p>
+                <p>Name: 20</p>
+                <p>Name: 50</p>
+                <p>Name: 78</p>
+                <p>Name: 125</p>
+                <p>Name: 77</p>
+                <p>Name: 42</p>
+            </div>
```

Comment:
- [ ] good job making your project look like the wireframe 👍 however according to the requirements, you should use HTML just to make the layout. To view the items inside the list you are expected to use ES6 modules, which includes the usage of arrow functions, import and export and proper usage of const and let keywords. as mentioned in [the assignment](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)

Reply:
Thank you